### ISSUE TITLE
Does not comply with build tags

### ISSUE THREAD

**Issue by alecthomas** [jmhodges/copyfighter#6]
When attempting to check a package with platform build tags, copyfighter errors with a duplicate symbol because it is not excluding files via the tags:

```
$ copyfighter .
unable to type check package "kingpin": guesswidth_unix.go:13:6: guessWidth redeclared in this block
$ head -1 guesswidth*
==> guesswidth.go <==
// +build appengine !linux,!freebsd,!darwin,!dragonfly,!netbsd,!openbsd

==> guesswidth_unix.go <==
// +build !appengine,linux freebsd darwin dragonfly netbsd openbsd
```

**Comment by kchr** [comment]
I'd also really like to see this fixed. Upvote!

**Comment by cristian-sima** [comment]
Same here!

**Comment by sni** [comment]
should be fixed with #14

**Comment by jmhodges** [comment]
Merged!
